ext/tags.go

```diff
@@ -19,30 +19,32 @@ var (
 	//////////////////////////////////////////////////////////////////////
 
 	// SpanKind hints at relationship between spans, e.g. client/server
-	SpanKind = spanKindTag("span.kind")
+	SpanKind = spanKindTagName("span.kind")
 
 	// SpanKindRPCClient marks a span representing the client-side of an RPC
 	// or other remote call
-	SpanKindRPCClient = SpanKindEnum("client")
+	SpanKindRPCClientEnum = SpanKindEnum("client")
```

Comment:
OK, I vote to eliminate the Enum values entirely. I really think this confusion is over whether "tag" refers to a tag name (one string) or a tag+value (string, interface{}).

Do you mind that SpanKindRPCServer and SpanKindRPCClient are in this file, and are not tag names but tag+values?